### ISSUE TITLE
The "source <(kapp completion bash)" command outputs an error message

### ISSUE THREAD

**Issue by itaytalmi** [k14s/kapp#447]
**What steps did you take:**
[A clear and concise description steps and any files that can be used to reproduce the problem.]

Added `source <(kapp completion bash)` to my `/etc/profile` so it would automatically load.

To reproduce the issue, you can simply run `source <(kapp completion bash)`.

**What happened:**
[A small description of the result]

Got the following error message:

```text
Succeeded: command not found
```

This is because the `kapp completion bash` commands outputs a `Succeeded` string at the end, which is not a valid command, and when used with `source`, bash tries to execute it.

Bash completion does work, though.

**What did you expect:**
[A description of what was expected]

**Anything else you would like to add:**
[Additional information that will assist in solving the issue.]

As a workaround, I'm using the following:

```bash
source <(kapp completion bash | grep -v Succeeded)
```

This removes the `Succeeded` string from the output, so the error doesn't occur.

Same issue also occurs with imgpkg, as I described here:
https://github.com/vmware-tanzu/carvel-imgpkg/issues/342

This issue doesn't occur with ytt, because `ytt completion bash` doesn't output the `Succeeded` string.

**Environment:**

- kapp version (use `kapp --version`): `0.46.0`
- OS (e.g. from `/etc/os-release`): Using `Linux Ubuntu 20.04.4 LTS (Focal Fossa)` and `macOS Monterey 12.2.1`, but this issue occurs on any bash.
- Kubernetes version (use `kubectl version`): Irrelevant

---
Vote on this request

This is an invitation to the community to vote on issues, to help us prioritize our backlog. Use the "smiley face" up to the right of this comment to vote.

👍 "I would like to see this addressed as soon as possible"
👎 "There are other more important things to focus on right now"

We are also happy to receive and review Pull Requests if you want to help working on this issue.

**Comment by renuy** [comment]
Thanks @itaytalmi  for bringing this up. Adding it to our un-prioritised backlog for now. 
We would be happy to prioritise a review, if you want submit a PR for the same.

**Comment by rohitagg2020** [comment]
Hi @itaytalmi 

kapp sets `--tty=true` by default which results in the extra "Succeeded" at the end of the file. 

We have added a fix in the `develop` branch. To avoid printing extra "Succeeded" in the end, add `--tty=false` flag to the kapp command. e.g. `source <(kapp completion bash --tty=false)`.

This fix will be available from the next `kapp` release i.e. `>=0.47.0`.
Once, a next kapp release is made, we will come and update the issue.

**Comment by renuy** [comment]
Fix available as part of kapp v0.47.0
